flink-streaming-connectors/flink-connector-filesystem/src/test/java/org/apache/flink/streaming/connectors/fs/bucketing/BucketingSinkFaultTolerance2ITCase.java

```diff
@@ -61,6 +62,10 @@
 * high. This provokes the case that the sink restarts without any checkpoint having been performed.
 * This tests the initial cleanup of pending/in-progress files.
 */
+
+// I suggest to remove this as we no longer clean up lingering files.
+
+@Ignore
 public class BucketingSinkFaultTolerance2ITCase extends StreamFaultToleranceTestBase {
```

Comment:
according to the javadocs we can indeed remove this test. Even without that, i can't quite figure out where the test actually verified that the files were deleted :/